opacclient/libopac/src/main/java/de/geeksfactory/opacclient/apis/BaseApi.java

```diff
@@ -472,6 +472,17 @@ public void setStringProvider(StringProvider stringProvider) {
         this.stringProvider = stringProvider;
     }
 
+    protected String buildHttpGetParams(Map<String, String> params,
+                                        String encoding) throws UnsupportedEncodingException {
```

Comment:
I see only one place where a value is replaced (in `searchGetPage`). It could also be implemented using `List<NameValuePair>` by rebuilding the parameters for `searchGetPage` from the `lastQuery` (adding a `page` parameter to `buildSearchParams`). Of course, this would slow down the `searchGetPage` function, but it is probably negligible compared to the HTTP request duration. And it would have the advantage of having one single function that generates the parameters for both `search` and `searchGetPage`.